packages/vite-plugin-react-router/package.json

```diff
@@ -54,11 +58,11 @@
   },
   "homepage": "https://github.com/netlify/remix-compute#readme",
   "dependencies": {
-    "isbot": "^5.0.0"
+    "@netlify/edge-functions": "^3.0.2",
+    "@netlify/functions": "^5.1.0",
```

Comment:
Not completely related, but these shouldn't be dev deps anymore, since they're needed for the `netlifyRouterContext` export added in 2.0.0.